docs/ops/doc/examples/labeling_mesh.rst

```diff
@@ -0,0 +1,244 @@
+======================================
+3D Labeling and Mesh Geometry Analysis
+======================================
+
+In this example we will use SciJava Ops to segment puncta only in the nuclear space and create individual
+meshes for each isolated puncta with geomtry measurements (*e.g.* sphericity). This example uses a dataset consisting of
+HeLa cells expressing HIV-1 :sub:`NL4-3` Vif protein (puncta) and stained with DAPI (4′,6-diamidino-2-phenylindole, nuclei).
+The dataset was collected on an epifluorescent microscope at 1.45NA 100x magnification (oil immersion) with 0.1 μm step size between
+slices.
+
+Using SciJava Ops to segment 3D puncta
+--------------------------------------
+
+The goal of this script is to segment and measure the 3D volume of the individual Vif puncta, but only within the nuclear space. In order
+to accomplish this analysis goal the script utilizes SciJava Ops to:
+
+1. Extract the selected channels from the open dataset.
+2. Performs puncta and nucleus specific image processing steps on the extract channels. *Note: these are intended as example, do your own clean up steps for your data instead!*
+3. Extracts a binary mask of only the puncta from the nuclear mask regions using logical operations (*i.e.* OR, AND, XOR, *etc*...).
+4. Create label images and meshes for each puncta within a nuclear region.
+5. Measure the 3D geometry of the puncta and nuclear regions.
+6. Display results in two tables (puncta and nuclear).
+
+To use the script in the example first download the sample dataset `here`_. Next, open Fiji, the sample image and this scipt. Click **Run** to create the script parameter GUI,
+where you can customize some values to your own data, such as the channel names, channel position and image calibration values.
+
+.. figure:: https://media.scijava.org/scijava-ops/1.0.1/labeling_mesh_example_dialog.png
+
+Once the script has been configured click **OK** to start the analysis. The script will display two tables (one for each channel with the given channel name) and the labeling
```

Comment:
This took longer than I expected to run - might not be a bad idea to give the users an idea of how long it will take, and to let them know they can see which Op is running via the `TaskService` (unfortunately, I don't think there's an imagej.net wiki page on the `TaskService`, is there @ctrueden?)

Additionally, some representation of the expected table outputs would be nice.

Reply:
I added a new section on the expected outputs (i.e. the index image and the tables) in 7836ba5fa622365b7dc7f2432ae5049fa5f1aad3.

Reply:
I've already removed the fill holes operation so this script is much faster now. Mark said he was able to get it down to under 30 seconds when he trimmed the data to only a few planes.